testing/scripts/Makefile

```diff
@@ -61,7 +63,8 @@ install:
 
 .PHONY: test
 test: build_protos install
-	pytest --verbose -s -W ignore 2>&1
+	pip install pytest-xdist
```

Comment:
We could add `pytest-xdist` as part of the `./testing/scripts/dev_requirements.txt` file.

Reply:
Good catch